style.css

```diff
@@ -10,7 +13,11 @@
     } 
     .img-portada {
         block-size: 23vh;
+        
     }
+    .imago{
+    block-size:15vh;
```

Comment:
💅 I think you could rename this class with something more descriptive and try to follow BEM nomenclature.
Also you have wrong indentation.

Reply:
ready